godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch

```diff
@@ -37,15 +48,33 @@
       <rosparam command="load" file="$(find godel_irb2400_support)/config/point_cloud_descriptions.yaml"/>
     </node>
   </group>
+
+  <!-- If the user uses the real sensor, launch the ensenso publisher node -->
+  <group unless="$(arg sim_sensor)">
+    <remap from="/depth/points" to="/sensor_point_cloud"/>
+    <param name="camera_frame_id" type="string" value="tool0" />
+    <node name="ensenso_publisher_node" pkg="ensenso_publisher" type="ensenso_publisher_node" output="screen">
+        <param name="FlexView" type="bool" value="true" />
+        <param name="FlexViewImages" type="int" value="4" />
```

Comment:
More long term, we should look into saving and restoring entire configurations from the Ensenso. We'll probably want to have "profiles" that we can load and apply for different part types.